VERSIONS.md

```diff
@@ -1,6 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 |---------|-------------|-----------------|----------------------|
-| 4.1.4   | 3.11.1      | 4.2.1           | 1.7.1                |
+| 4.2.0   | 3.11.1      | 4.2.1           | 1.7.1                |
```

Comment:
It should be a new line with 4.2.0, without removing 4.1.4

Reply:
🤦